docs/source/docker/advanced/datapool.rst

```diff
@@ -67,6 +67,27 @@ first time.
     |-- passageway1-c1.avi
     `-- terrace1-c0.avi
 
+Let's create a Lightly dataset which uses that bucket (choose your tab - S3, GCS or Azure):
+
+.. tab-set::
+  .. tab-item:: AWS S3 Datasource
```

Comment:
Could you use `.. tabs::` and `... tab::` instead of `.. tab-set::` and `.. tab-item::` just to be consistent with the rest of the docs.

Reply:
done